cmds/webboot/webboot.go

```diff
@@ -227,6 +228,16 @@ func main() {
 			cacheDir = cachePath
 		}
 	}
+
+	if err := menu.Init(); err != nil {
```

Comment:
could menu.Init ever have error messages you want to buffer too?

Reply:
I don't think so in this case. `menu.Init()` really just wraps `termui.Init()`, and [the only return value](https://github.com/gizak/termui/blob/master/v3/backend.go#L13) is an error. 

I think if the UI doesn't init, the program should terminate.